src/tests/test_salute.py

```diff
@@ -0,0 +1,35 @@
+import os
+import json
+import pytest
+
+from ..service.salute import SaluteService
+
+@pytest.fixture(name="salute", autouse=True)
+def fixture_salute():
+    return SaluteService()
+
+@pytest.fixture(name="file_salute", autouse=True)
+def fixture_file_salute():
+    filename = os.path.join(os.path.dirname(__file__), "../json/salutes.json")
```

Comment:
This will need to be updated after you move this to the new `src/tests/service` directory
```suggestion
    filename = os.path.join(os.path.dirname(__file__), "../../json/salutes.json")
```